mongodbatlas/data_source_mongodbatlas_advanced_cluster_test.go

```diff
@@ -1,148 +0,0 @@
-package mongodbatlas
-
-import (
-	"fmt"
-	"os"
-	"testing"
-
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/mwielbut/pointy"
-	matlas "go.mongodb.org/atlas/mongodbatlas"
-)
-
-func TestAccClusterDSAdvancedCluster_basic(t *testing.T) {
-	var (
-		cluster        matlas.AdvancedCluster
-		resourceName   = "mongodbatlas_advanced_cluster.test"
-		dataSourceName = "data.mongodbatlas_advanced_cluster.test"
```

Comment:
I have moved the data source tests into the resource tests. By doing this, there is no need to create new clusters specifically for these tests. Instead, we can efficiently reuse the clusters that are already created for the resource tests.